virtool/analyses/db.py

```diff
@@ -208,6 +214,71 @@ async def create(
     return document
 
 
+async def create_row(pg: AsyncEngine, analysis_id: str, analysis_format: str, name: str) -> Dict[str, any]:
```

Comment:
The naming of these is going to clash with the `Analysis` model when it is eventually created.

I think we should move the analysis file code into a new module: `virtool.analyses.files`. Also, change these function names to something like `create_analysis_file`.